packages/cursorless-engine/src/cursorlessEngine.ts

```diff
@@ -101,13 +104,15 @@ export function createCursorlessEngine(
       customSpokenFormGenerator,
     ),
     customSpokenFormGenerator,
-    testCaseRecorder,
     storedTargets,
     hatTokenMap,
     snippets,
     injectIde,
     runIntegrationTests: () =>
       runIntegrationTests(treeSitter, languageDefinitions),
+    addCommandRunnerDecorator: (decorator: CommandRunnerDecorator) => {
+      commandRunnerDecorators.push(decorator);
+    },
```

Comment:
Feels odd not to return a disposable that can be used to remove the decorator. But I guess we don't have a use case for it so probably fine 🤷‍♂️

Reply:
Is there unit test infrastructure so I could add, e.g., a test to confirm that decorators added this way are actually invoked?  I ask because if I added removal I'd definitely want to test that, but of course that just points out the lack of tests for calling the decorators in the first place.

Reply:
We have end-to-end tests for the test case recorder, so if your decorators weren't working, those tests would break. Does that work?

Reply:
Sure, that's fine. I just wanted to make sure I wasn't leaving anything
out. And I'm happier not writing the dispose code given lack of use case or
tests.